nest/06.glossary.md

```diff
@@ -0,0 +1,182 @@
+# 6 Glossary
+
+## A
+
+## B
+
+## C
+[//]: # (TODO: add link to the section 3.2)
+* Cache [section 3.2]()
+
+In Nest, the cache is a local copy of the list of available [packages]() from Raven-OS server.
+The cache has to be kept up-to-date in order to install the latest versions of the packages.
+
+* Category
+
+[//]: # (TODO: add links to the 'repository' and 'package' entry of this glossary)
+Inside each [repository](), there are several categories.
+The [packages]() are split between the repositories and their categories.
+Each package name must be unique in a category of a repository, but there can be several packages with the same name spread in several categories.
+
+## D
+[//]: # (TODO: add link to the section 1.2)
+* Dependency [section 1.2]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+A dependency is a [package]() A that is required by a package B, so that the package B can function properly.
+
+[//]: # (TODO: add link to the section 4.1.3)
+* Dependency graph/Dependency tree [section 4.1.3]()
+
+[//]: # (TODO: add link to the 'package' entry of this glossary)
+The dependency graph is a directed graph (i.e. a graph with directed connections) representing the dependencies between [packages]().
+## E
+
+## F
+[//]: # (TODO: add link to the chapter 4)
+* `finest` [chapter 4]()
+
+`finest` is the binary of Nest targeted towards experienced users.
+
+* `finest group create <name> [--parent <group>]`
```

Comment:
I'm not sure commands should be in the glossary. What was your idea behind that?

Reply:
the glossary seemed empty, so I wanted to add more things to it D:
but I agree, they shouldn't be there, I'll remove them